src/fparser/statements.py

```diff
@@ -641,13 +642,21 @@ class Allocate(Statement):
     match = re.compile(r'allocate\s*\(.*\)\Z', re.I).match
 
     def process_item(self):
+        '''
+        Process the allocate statement and store the various entities being
+        allocated in self.items. Any type-specification is stored in
+        self.spec.
+
+        :raises ParseError: if an invalid type-specification is used
+        '''
         line = self.item.get_line()[8:].lstrip()[1:-1].strip()
         item2 = self.item.copy(line, True)
         line2 = item2.get_line()
         i = line2.find('::')
         if i != -1:
             spec = item2.apply_map(line2[:i].rstrip())
             from .block_statements import type_spec
+            from .utils import is_name
```

Comment:
This returns pylint error as from .utils import is_name is already defined at the top of file. Please remove.